apps/aevm/src/aevm_ae_primops.erl

```diff
@@ -197,6 +197,8 @@ types(?PRIM_CALL_SPEND,_HeapValue,_Store,_State) ->
     {[word], tuple0_t()};
 types(?PRIM_CALL_CRYPTO_ECVERIFY, _HeapValue, _Store, _State) ->
     {[word, word, sign_t()], word};
+types(?PRIM_CALL_CRYPTO_ECVERIFY_SECP256K1, _HeapValue, _Store, _State) ->
```

Comment:
I need to check impact of this.

Reply:
I see this function is called from [`aevm_eeevm_state:get_contract_call_input`](https://github.com/aeternity/aeternity/blob/27f6eb290a228d93b48ef7ac219539cbf13139df/apps/aevm/src/aevm_eeevm_state.erl#L432).

Reply:
It is fine - out of gas error on Minerva protocol before and after this change.